stress_test.py

```diff
@@ -0,0 +1,413 @@
+import argparse
+import os
+import random
+import string
+import sys
+import time
+
+from enum import Enum
+from multiprocessing import cpu_count, Process, Pool
+from threading import Thread
+
+from test_cli import run_command
+
+"""
+Script to stress test CodaLab's backend. The following is a list of what's being tested:
+- Large bundle upload
+- Many small bundle uploads
+- Many worksheet creations and copies
+- Many small runs (trivial commands) in parallel
+- Run that writes infinitely to disk
+- Many runs that all use up plenty of disk (1GB each)
+- Run that uses infinite memory
+- Run that tries to use up all the GPUs (+ memory)
+- Many runs that use all sorts of large Docker images
+- Call CodaLab repeatedly to get information to check if the system is still responsive
+"""
+
+
+class StressTestArg(Enum):
```

Comment:
This is not necessary since `argparse` does it for you...you can `add_argument('--large-file-size-gb', ...)` and then access the arguments with `args.large_file_size_gb`.  This should simplify things quite a bit.